src/exported.ts

```diff
@@ -5,8 +5,11 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import { Flags } from '@oclif/core';
```

Comment:
Why does this need to be exported?

Reply:
@mdonnalley For connivence only. If this is something we want to shy away from, reexporting base class "stuff, each subclass can refer to oclif/core